helx/charts/nginx/values.yaml

```diff
@@ -52,6 +52,12 @@ ingress:
   tls:
     # -- Values inserted into the TLS block come from SSL.nginxTLSSecret and service.serverName for backward compatibility
     enabled: true
+  admin:
+    # Whether to create an additional Ingress to restrict access to /admin routes
+    enabled: false
+    annotations: {}
```

Comment:
Looks like these annotations aren't being included in the admin ingress (since it only includes .Values.ingress.annotations and not .Values.ingress.admin.annotations).